catalog/src/main/kotlin/com/adevinta/spark/catalog/configurator/samples/textfields/DropdownsConfigurator.kt

```diff
@@ -102,14 +107,34 @@ private fun ColumnScope.DropdownSample() {
         visualTransformation = VisualTransformation.None,
         interactionSource = remember { MutableInteractionSource() },
     ) {
-        repeat(5) {
-            DropdownMenuItem(
-                text = { Text("Item $it") },
-                onClick = { expanded = false },
-            )
+        DropdownStubData.forEach { (groupName, books) ->
+            DropdownMenuGroupItem(
+                title = {
+                    Text(groupName)
+                },
+            ) {
+                books.forEach { book ->
+                    DropdownMenuItem(
+                        text = { Text(book) },
+                        onClick = {
+                            valueText = book
+                            if(hideOnSelect) expanded = false
```

Comment:
You'll probably have to re-run spotless or manually reformat the files
```suggestion
                            if (hideOnSelect) expanded = false
```